vms/volunteer/tests/test_services.py

```diff
@@ -18,12 +20,12 @@
 class VolunteerMethodTests(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        country_name = 'India'
-        country = get_country_by_name(country_name)
-        state_name = 'Uttarakhand'
-        state = get_state_by_name(state_name)
-        city_name = 'Roorkee'
-        city = get_city_by_name(city_name)
+        # country_name = 'India'
```

Comment:
Why change?

Reply:
Same reason for all the `create_` functions used, these were not present in the database.
And due to This Not Found error was thrown.

Reply:
Check it on CI first, locally it might be due to the download of cities_light

Reply:
Yes, I checked on CI.
I think `cities_light` is not used in tests and does not affect tests.
That's why `create_` functions were previously created. (I didn't write `create_` functions I only called them.)